source/upgrade.txt

```diff
@@ -61,8 +61,10 @@ the version after v4.0 including any listed under v4.5.
 Version 5.0 Breaking Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- This driver version introduces the following changes to the ``ConnectionId`` class:
+This driver version introduces the following breaking changes:
```

Comment:
The introduction of this phrasing required minor changes to the beginning of several bullet points.